### DISCUSSION TITLE
Preserving Airtable record order during import [Apr 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5526] [Discord Threads]
> 🙋‍♂️ **saurav_mitra**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### There is no direct way currently. Unless you have some filter OR sort enabled.   
####    
#### If there are not many tables, there is a work around.   
#### - add an "Autonumber" column on your airtable table (say ID)   
#### - after import, sort by 'ID' in nocodb  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### this has been an open concern. Data retrieved via Airtable APIs need not be in order as seen on UI. Below is a discussion, but for column ordering   
#### https://community.airtable.com/t5/development-apis/colums-out-of-order/td-p/57789  
#  
> 🙋‍♂️ **saurav_mitra**  
#### Yeah, I'll probably go with the autonumber solution, or just sort by creation time. Thanks for the quick answer 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1086256974896844940)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
